src/main/java/com/sap/oss/phosphor/fosstars/data/github/GithubInnerSourceTopicDataProvider.java

```diff
@@ -0,0 +1,54 @@
+package com.sap.oss.phosphor.fosstars.data.github;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.HAS_INNERSOURCE_TOPIC;
+
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.other.Utils;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.GitHubProject;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.Set;
+
+public class GithubInnerSourceTopicDataProvider extends GitHubCachingDataProvider {
+
+  private static final Set<String> ACCEPTED_TOPICS;
+
+  static {
+    Set<String> acceptedTopics = new HashSet<>();
+    acceptedTopics.add("inner-source");
+    acceptedTopics.add("innersource");
+    ACCEPTED_TOPICS = Collections.unmodifiableSet(acceptedTopics);
+  }
```

Comment:
```suggestion
  private static final Set<String> ACCEPTED_TOPICS = new HashSet<>(Arrays.asList("inner-source", "innersource"));
```

Reply:
No, `static final` would make (a) a class level variable and (b) a final reference to a Set. The Set itself would still be modifiable.

Reply:
Yes you are right, I misread this. Thanks for clarifying.